chatbees_script.js

```diff
@@ -113,9 +118,12 @@ function chatbeesSendMessage() {
   })
   .catch(error => {
       console.error('Error:', error);
-      // Display the error in the chat area
+      // remove the thinking message
+      chatArea.removeChild(thinkMsg);
+
+      // Display a generic message for error case
       var botMsg = document.createElement('div');
-      botMsg.textContent = error;
+      botMsg.textContent = "Something went wrong, please try again later";
```

Comment:
why not show the error? if customer reports issue, we will have to ask them to check console log, which may be hard for non-tech people, also not easy on mobile.

Reply:
sg, appended the error message